mibidata/tiff.py

```diff
@@ -31,6 +31,9 @@
 # Encoding of tiff tags.
 ENCODING = 'utf-8'
 
+REQUIRED_METADATA_ATTRIBUTES = ('fov_id', 'fov_name', 'run', 'folder',
```

Comment:
As it stands, this isn't used anywhere, though see my next comment before deciding to remove.